english/showerthoughts/showerthoughts.md

```diff
@@ -0,0 +1 @@
+[showerthoughts](https://www.reddit.com/r/Showerthoughts/)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.